docs/operator-manual/ui-extension/tracing.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_label: "Tracing"
+title: ""
+---
+
+Tracing allows to collect fine grained details about policy evaluations. It can be a useful tool for debugging issues inside of your Kubewarden deployment and policies.
+
+We will use [Jaeger](https://www.jaegertracing.io/) -- used to receive, store and visualize trace events.
+
+## Prerequisites
+
+Cert-Manager and OpenTelemetry are required.
+Follow [these instructions](../telemetry/opentelemetry/01-quickstart.md#install-opentelemetry) to install Cert Manager and the OpenTelemetry Operator.
+
+## Install Jaeger
+
+> Adapted from the [tracing quickstart](../telemetry/tracing/01-quickstart.md)
```

Comment:
I would actually prefer to just point to the other page, to not duplicate content that may get out of sync.
If there's no installation specifics to the UI I think it's fine.

Reply:
I agree on that

Reply:
I updated that section to link to the other page, left a few things in there that felt relatable to the UI.